packages/runtime/src/runtime.ts

```diff
@@ -226,6 +226,15 @@ export class Runtime {
 		}
 	}
 
+	validateNoDupTxn(gtxns: EncTx[]): void {
```

Comment:
Im thinking if `assertNoDuplicateTransaction` name wouldnt be better. Let me know what do you think

Reply:
I updated it, please check PR again!